assets/components/src/card/style.scss

```diff
@@ -21,11 +20,8 @@
 	&__buttons-card {
 		display: flex;
 		flex-wrap: wrap;
-		margin: 32px -8px 16px;
```

Comment:
I think removing these styles causes some strange alignment in this part of the Components Demo page. Not sure if it affects any real UI, though:

<img width="392" alt="Screen Shot 2022-06-09 at 12 56 26 PM" src="https://user-images.githubusercontent.com/2230142/172926733-06d1266c-5495-494a-93ac-4fc0c8753c28.png">

Reply:
This specific demo was always misaligned.
![Screenshot 2022-06-10 at 10 06 11](https://user-images.githubusercontent.com/177929/173031906-85d3e79f-f1d1-4489-81fc-b0f00f436378.png)

Anyway... I fixed the alignment: https://github.com/Automattic/newspack-plugin/pull/1676/commits/983b3f2f0546b3823389598454f56d4eb5b303ce 😄